src/MSBuild/Resources/Strings.resx

```diff
@@ -929,6 +929,120 @@
+                      (Short form: -tl)
+    </value>
+    <comment>
+      LOCALIZATION: "-terminallogger" and "-tl" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_51_GetResultOutputFileSwitch" Visibility="Public">
+    <value>  -getResultOutputFile:file
+                     Redirect output from get* into a file.
+
+                     Example:
+                     -getProperty:Bar -getResultOutputFile:Biz.txt
+                     This writes the value of property Bar into Biz.txt.
+    </value>
+    <comment>
+      LOCALIZATION: "-getResultOutputFile" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_48_TerminalLoggerParametersSwitch" Visibility="Public">
+    <value>  -termialLoggerParameters:default=[auto|false|true]
+                      Users are able to opt-in and use the TerminalLogger.
+                      - `true` forces TerminalLogger to be used even wwhen it
+                      would be disabled.
+                      - `false` forces TerminalLogger to not be used even when
+                      it would be enabled.
+                      - `auto` enables TerminalLogger when the terminal supports
+                      it and the session doesn't have redirected stdout/stderr
+                      (Short form: -tlp)
+                      Example:
+                        -termialLoggerParameters:default=auto
+    </value>
+    <comment>
+      LOCALIZATION: "-termialLoggerParameters"  and "-tlp" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_49_WaitForDebuggerSwitch" Visibility="Public">
+    <value>  -waitfordebugger
+                      Wait for debugger command line switch in Debug build.
+                      (Short form: -wfd)
+    </value>
+    <comment>
+      LOCALIZATION: "-waitfordebugger" and "-wfd" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_50_NodeModeSwitch" Visibility="Public">
+    <value>  -nodemode:number
+                      It's used when MSBuild spawns a process, indicating that
+                      MSBuild should start up in "Out of process worker node"
+                      mode.
+                      (Short form: -nmode)
+                      Example:
+                        -nodemode:1
+    </value>
+    <comment>
+      LOCALIZATION: "-nodemode" and "-nmode" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
```

Comment:
This switch may be better left undocumented as an exception. Tagging @baronfel for his opinion.

Reply:
Are users ever expected to manage their own node pool directly like this? I can't think of a reason for them to do so. I'd be ok with hiding this from the user-facing help docs (though the in-repo/detailed developer docs should cover it).

Reply:
https://github.com/dotnet/msbuild/blob/494047fc6f53e60048d23f0cc2350a042ba02526/src/MSBuild/CommandLineSwitches.cs#L246 has the short form

Reply:
Remove the -nodemode helpmessage